### ISSUE TITLE
Sohpizer : Cannot resolve template loader path [classpath:/templates/]to [java.io.File]: using SpringTemplateLoader as fallback

### ISSUE THREAD

**Issue by anandaraj3121** [shopizer-ecommerce/shopizer#177]
Hi Shopizer Team,

I was trying to deploy the shopizer WAR in a standalone tomcat server, But the deployment getting failed due to "java.io.FileNotFoundException: class path resource [templates/] cannot be resolved to absolute file path because it does not reside in the file system".

Note : The same is working fine, when i try to run through spring-boot.
Kindly do the need full.

Log Trace:  [errorlog.txt](https://github.com/shopizer-ecommerce/shopizer/files/1367050/errorlog.txt)

Regards
Ananda RR (anandaraj3121@mail.com)

**Comment by anandaraj3121** [comment]
I think this is the duplicate of issue #159 . Any recent update on this ?
Because due this error, on trying to access the application, browser says 404.

**Comment by shopizer-ecommerce** [comment]
Yes templates issue is a known issue that does not prevent server to start

The log attached is not complete, i can't see why you cannot run the app

Preferably start compile package and run from maven first

mvn clean install
cd sm-shop
mvn spring-boot:run

**Comment by anandaraj3121** [comment]
Hi Thanks for the response;

Am able to successfully run the application through MVN SPRING-BOOT:RUN; Where as deployment on tomcat failing. In the tomcat log i could see ShopApplication started successfully. But on invoking the admin/login.html browser shows 404.

I use

Shopizer 2.0.5
JDK 1.8
maven 3.5.0
tomcat 8

Could you please suggest, whether i need to changes the version of any the above mentioned tools ?
Kindly do the need full, we are trying to deploy the shopizer application in a real web container.

**Comment by shopizer-ecommerce** [comment]
Hi

Here is the way i did it for some of the deployment

cd shopizer
mvn clean install
cd sm-shop/target

mv sm-shop-XYZ.war ROOT.war

put ROOT.war directly under tomcat webapps folder

http://<url>:8080
http://<url>:8080/admin
